tsconfig.json

```diff
@@ -7,8 +7,9 @@
     "moduleResolution": "node",
     "resolveJsonModule": true,
     "noImplicitAny": false,
+    "strictNullChecks": false,
```

Comment:
Remove this settings.
We do want to enforce strict null checks.
It probably means you would also need to fix some typing.